run-pipeline-on-rhoai-.md

```diff
@@ -0,0 +1,730 @@
+  * A data science project/namespace, in this document this will be referred to as `<data-science-project-name/namespace>`
+* A [StorageClass] that supports dynamic provisioning with [ReadWriteMany] access mode (see step 3 below).
+* An S3 object store such as AWS, or an alternative object storage solution that is S3-compliant such as Ceph, Nooba and MinIO.
+* A locally installed `oc` command line tool to create and manage kubernetes resources.
+* Ilab CLI (or Skopeo/Oras/etc.) for model downloads
+* For Disconnected Clusters:
+  * **Mirror Required Images**:
+
+    In a disconnected environment, you must mirror the following container images to your internal registry before running the pipeline. Use tools like `oc adm release mirror`, `skopeo`, or `oras` to mirror these images:
+    - `registry.redhat.io/ubi9/toolbox@sha256:da31dee8904a535d12689346e65e5b00d11a6179abf1fa69b548dbd755fa2770`
+    - `registry.redhat.io/openshift4/ose-cli@sha256:1d5c8442a6ec745e6ae44a7738c0681f1e21aac8be76ba826c2ddf2eed8475db`
+    - `registry.redhat.io/rhelai1/instructlab-nvidia-rhel9@sha256:b3dc9af0244aa6b84e6c3ef53e714a316daaefaae67e28de397cd71ee4b2ac7e`
+    - `registry.redhat.io/rhelai1/skills-adapter-v3@sha256:53dd11a762bb39fc33c15499891309f0cdc8dbfd02abf94c9c60aad643aca255`
+    - `registry.redhat.io/rhelai1/knowledge-adapter-v3@sha256:ef1608ec78d5e39655b505544c0f30a015a6c9cb7e2b2deffe394791f8c76c6f`
+    - `registry.redhat.io/rhelai1/mixtral-8x7b-instruct-v0-1@sha256:bc08e466aa35352a621d0ad221c2e247ff9751f4cb6cffe00d5894ce6bfd3fd7`
+    - `registry.redhat.io/rhelai1/prometheus-8x7b-v2-0@sha256:9fcb49c230f6e73ff944377307bb83a05ae3ac20300af75e429151f4f8bf4285`
+    - `quay.io/modh/odh-generic-data-science-notebook@sha256:7c1a4ca213b71d342a2d1366171304e469da06d5f15710fab5dd3ce013aa1b73`
+    - `quay.io/modh/vllm@sha256:3c56d4c2a5a9565e8b07ba17a6624290c4fb39ac9097b99b946326c09a8b40c8`
+    - `quay.io/modh/vllm@sha256:97b91f9bd71202f5de8d379cfb61baec887b47f836a2ff8b158c946196de5660`
+    - `quay.io/opendatahub/workbench-images@sha256:7f26f5f2bec4184af15acd95f29b3450526c5c28c386b6cb694fbe82d71d0b41`
+    - `ghcr.io/oras-project/oras:main@sha256:8859e7e3ae510fb921ebeb109ac9d3e3bb91799e0d52001ae456df33929029db`
+
+  * **500GB PersistentVolumeClaim (PVC) for Mixtral**:
+
+    The proposed method to deploy Mixtral requires a **500GB PVC**.
+    - In a **disconnected cluster**, ensure that your OpenShift environment has sufficient storage capacity and a **StorageClass** configured to provision this PVC.
+    - If automatic PVC creation fails, you may need to manually create a PersistentVolume (PV) and bind it to a PVC.
+  * **Additional Git Repository for Mirroring**:
+
+    - The iLab pipeline uses a taxonomy repo from GitHub.
+    - The following repository should be mirrored for use in a disconnected cluster:
+
+      ```bash
+      Original repo: github.com/sallyom/taxonomy.git
+      repo_branch: utah-hockey
+      ```
```

Comment:
Sorry, this was just a note in our internal document. Sally's repo was just an example we used
```suggestion
  * **Accessible git repository with the taxonomy**:

    - The iLab pipeline uses a taxonomy git repository, which should be accessible from the disconnected cluster
```